htdocs/accountancy/class/bookkeeping.class.php

```diff
@@ -1875,7 +1875,7 @@ public function get_compte_racine($account = null)
 				$obj = $this->db->fetch_object($resql);
 			}
 
-			return $obj->label;
+			return $obj;
```

Comment:
A function (except the method query of database handler) should not return a result set $obj.
This result in a memory pointer open inside a function and that are used and must be closed outside of function.
Can you instead return an array with data you need.

Reply:
Why returning an array with 2 data, when the first data is always same than the entry parameter ?
Returning the second data only is enough since the caller already known the first data